dev_run_local_4.x.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# as system admin, delete everything that matters
+oc login -u system:admin
+
+# this tells CVO not to reconcile the operator deployment.
+# this is necessary if you want to run the operator binary locally
+# and not fight with the existing deployment.
+# otherwise, CVO will continually re-deploy the operator deployment
+# every time you delete it and the local operator will have to fight
+# with it.  You may still be able to "win" as the configmap console-operator-lock
+# ensures via leader election that only one of the operators will
+# act at a time, but its best to avoid this hassle.
+oc apply -f ./dev/cvo-disable-operator.yaml
```

Comment:
It would be better to use `oc patch` to add the one override.

Reply:
Ok will do.